File: ts-sdk/src/sdk/models/operations/getmev2.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import * as shared from "../shared";



export class GetMeV2Response extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  userV2?: shared.UserV2;
}

File: user-api/python-sdk/src/epilot/models/operations/getmev2.py
import dataclasses
from typing import Optional
from ..shared import userv2 as shared_userv2


@dataclasses.dataclass
class GetMeV2Response:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    user_v2: Optional[shared_userv2.UserV2] = dataclasses.field(default=None)
    
